packages/sei-cosmwasm/src/sei_types.rs

```diff
@@ -27,6 +27,19 @@ pub enum OrderStatus {
     Fulfilled = 3,
 }
 
+#[derive(Serialize, Deserialize, Copy, Clone, Debug, PartialEq, JsonSchema, Eq, Hash)]
+pub enum PositionEffect {
+    Unknown,
+    Open,
+    Close,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct OrderData {
```

Comment:
`OrderData` may vary by contract. This definition is particular to Vortex, so I don't think it's appropriate to have it in `sei-cosmwasm`. It can be in `vortex-common`. As for the tester contract, since we definitely don't want it to have to depend on `vortex-common`, we can just redefine it there and add a comment saying that this `OrderData` definition is just an example.

Reply:
make sense, will do